jekyll/_includes/snippets/pipeline-values.adoc

```diff
@@ -0,0 +1,55 @@
+[.table.table-striped]
+[cols=3*, options="header", stripes=even]
+|===
+| Variable
+| Type                      
+| Value
+
+| `pipeline.id`                 
+|
```

Comment:
I need help with the types in this tables if possible

Reply:
Pipeline number is an integer.

in_setup is a boolean.

The rest is strings of various formats (UUIDs, hashes, etc.).